tasks/install_shell.sh

```diff
@@ -304,6 +310,22 @@ capture_tmp_stderr() {
 
 trap "rm -f $tmp_stderr; rm -rf $tmp_dir; exit $1" 1 2 15
 
+retry_cmd() {
+  a=0
+  while test $a -lt $retry; do
+    log "Retrying... [$((a + 1))/$retry]"
+    eval $1
+    rc=$?
+    info $rc
```

Comment:
How about something like `info "Return code: ${rc}"` and moved after checking if it's 0 so we get prints only about failing commands?